node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -45,30 +44,117 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
 
   implicit def executionContext: ExecutionContext = system.dispatcher
 
-  val peers: Vector[Peer]
-
-  private val _peerServices: mutable.Map[Peer, ServiceIdentifier] =
+  private val _peerData: mutable.Map[Peer, PeerData] =
     mutable.Map.empty
 
-  def peerServices: Map[Peer, ServiceIdentifier] = _peerServices.toMap
+  def peerData: Map[Peer, PeerData] = _peerData.toMap
+
+  def peers: Vector[Peer] = peerData.keys.toVector
+
+  /** Returns peers by querying each dns seed once. These will be IPv4 addresses. */
+  def getPeersFromDnsSeeds: Vector[Peer] = {
```

Comment:
So i think this is going to be a problem. If a user has tor enabled, will this network request be done over clearnet or tor?

Reply:
Over clearnet. I suppose we would need to change that. Will look into it.